### ISSUE TITLE
Cryptopia Sell Order function returns undefined id

### ISSUE THREAD

**Issue by rhrahul** [ccxt/ccxt#3065]
Hi,

here is the method I am using,
`cryptopia.create_limit_sell_order(pair, sell_quantity, sell_price)`

and this is output.
![output_cryptopia_sell](https://user-images.githubusercontent.com/35340981/41062453-7dcc01ea-69f3-11e8-8774-46530236eeb4.png)

- OS: Windows 10
- Programming Language version: Angular 4
- CCXT version: Latest
- Exchange: Cryptopia
- Method: create_limit_sell_order()

**Comment by kroitor** [comment]
If you could just provide the verbose output – it would be of great help. Namely, a verbose output would contain the structure of FilledOrders, which is not shown on your image. Can you please follow the rules precisely? https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue. Or, if you can't, then please click the "Data" there and make another screenshot... This way we would be able to fix the reason of the issue rather quickly.

**Comment by kroitor** [comment]
```JavaScript
// JavaScript

cryptopia.verbose = true
cryptopia.create_limit_sell_order(pair, sell_quantity, sell_price)
```

- Then go to web-inspector console and copy all text from there to here.
- No images, just the entire verbose request and response.
- Without your keys, remove them from there.

**Comment by rhrahul** [comment]
Ok, sure

**Comment by rhrahul** [comment]
ohh, I think I found issue,
the order got filled so it returned undefined id, is that correct ??

![output_cryptopia_sell_2](https://user-images.githubusercontent.com/35340981/41063743-229f07fa-69f7-11e8-83e8-e3e87358b783.png)

as you can see in Data > FilledOrders

**Comment by kroitor** [comment]
> the order got filled so it returned undefined id, is that correct ??

Yes, however, because the API of Cryptopia is very poor, they don't return an order ID in that case, and they don't return the filled amounts either, so we don't know the proper status of the order either. So your only option is to call fetchOpenOrders next to see if the order is filled. 

This is described here:
- https://github.com/ccxt/ccxt/wiki/Manual#orders-cache

I've posted this to their forum, but they didn't reply, and many users reported having the same issues. Now they closed their forum completely. So, I guess, your only option now if to read CCXT Manual very carefully, as most of the above is already described there )

Hope this answers your questions. Closing this for now if you don't mind.
